libse/Utilities.cs

```diff
@@ -24,7 +24,7 @@ public static class Utilities
         /// <summary>
         /// Cached environment new line characters for faster lookup.
         /// </summary>
-        public static readonly char[] NewLineChars = Environment.NewLine.ToCharArray();
+        public static readonly char[] NewLineChars = { '\r', '\n' };
```

Comment:
I know, but the thing is even for **unix** user **NewLineChars* isn't going to 'cause problem